samples/go/Dockerfile

```diff
@@ -2,6 +2,12 @@ FROM golang:alpine
 WORKDIR /src
 COPY . .
 
+RUN apk update && apk add ca-certificates && rm -rf /var/cache/apk/*
```

Comment:
It is a good practice to keep the RUNs to a minimum to minimize the layers in the image.

Reply:
I understand, but I'm not sure how I can make them fewer.
We need to install the ca package stuff, then copy over the .crt over and update the certificates.
I don't think I can use `cp` in the first command, instead of the docker's `COPY`, since I'm copying from the source to the actual image, correct?